src/main/java/org/myrobotlab/service/TestCatcher.java

```diff
@@ -249,7 +241,7 @@ public void checkMsg(long timeout, String method, Object... checkParms) throws I
    * "would have" been created to make this direct callback
    * 
    */
-  public void addData(String method, Object... parms) {
+  public void addDatax(String method, Object... parms) {

```

Comment:
If this method isn't used anymore, it should be removed.

Reply:
aye .. agreed - found a unified way to check for appropriate messages.
I've removed it locally.